qiskit_experiments/framework/experiment_data.py

```diff
@@ -1254,59 +1364,65 @@ def analysis_results(
                     * int: Specific index of the analysis results.
                     * slice: A list slice of indexes.
                     * str: ID or name of the analysis result.
+
             refresh: Retrieve the latest analysis results from the server, if
                 an experiment service is available.
             block: If True block for any analysis callbacks to finish running.
             timeout: max time in seconds to wait for analysis callbacks to finish running.
+            verbosity: Level of verbosity of returned data table (1, 2, 3):
+
+                    * 1 (minimum): Return data from the analysis.
+                    * 2 (normal): With supplemental data about experiment.
+                    * 3 (finest): With extra data to communicate with experiment service.
```

Comment:
I think it would be more useful to the user if the extra fields included in levels 2 and 3 are listed here as well.

Reply:
If I understand your question correctly, the _extra field_ of `AnalysisResultData.extra` will be shown in every level. The _extra data_ in level3 indicates ids, which doesn't represent any result of the experiment (and ids are not necessary unless user save the data to result db, e.g. community users).